src/qana/QservRestrictorPlugin.cc

```diff
@@ -337,15 +337,15 @@ std::shared_ptr<query::SecIdxInRestrictor> makeSecondaryIndexRestrictor(query::I
 }
 
 /**
- * @brief Make a Secondary Index comparison restrictor for the given between predicate, if one of the
+ * @brief Make a Secondary Index 'between' restrictor for the given between predicate, if one of the
  *      columns in the predicate is a director column.
  *
  * @param betweenPredicate
  * @param context
  * @return std::shared_ptr<query::SIBetweenRestr> The restrictor that corresponds to the given predicate if
  * one of the columns is a director column, otherwise nullptr.
  */
-std::shared_ptr<query::SecIdxBetweenRestrictor> makeSecondaryIndexRestrictor(
+__attribute__((unused)) std::shared_ptr<query::SecIdxBetweenRestrictor> makeSecondaryIndexRestrictor(
```

Comment:
Since Qserv is officially at C++17 then you might consider a more portable syntax for this:
```
[[maybe_unused]]
```
see: https://en.cppreference.com/w/cpp/language/attributes/maybe_unused